docs/en/sql-reference/functions/other-functions.md

```diff
@@ -2109,6 +2300,14 @@ This function is not available in ClickHouse Cloud.
 Evaluate an external catboost model. [CatBoost](https://catboost.ai) is an open-source gradient boosting library developed by Yandex for machine learning.
 Accepts a path to a catboost model and model arguments (features). Returns Float64.
 
+**Syntax**
+
+```sql
+catboostEvaluate(path_to_model, feature_1, feature_2, …, feature_n)
```

Comment:
Maybe your editor changed `...` (three dots) into `…` (a single three-dot character), but we should use the former form. Please also grep if the latter symbol is used elsewhere in the docs and convert it to `...`, thanks.